#パーティクルの数
N: 600
#動作ノイズ
alpha1: 0.2
alpha2: 0.2
alpha3: 0.6
alpha4: 0.2
#レーザーノイズ
z_hit: 0.7
z_rand: 0.3
sigma_hit: 0.3
#レーザーパラメータ
laser_likelihood_max_dist: 2.0
max_beam: 90
MAX_RANGE: 25
MIN_RANGE: 0.02
#ランダムパーティクル
alpha_slow: 0.002
alpha_fast: 0.1
#初期位置、分散
init_x: -1.0
init_y: -3.2
init_theta: -0.3
init_x_cov: 0.3
init_y_cov: 0.3
init_theta_cov: 0.2
#しきい値
x_cov_thresh: 0.10
y_cov_thresh: 0.10
motion_update: 0.2
angle_update: 0.1
#rvizから初期位置を取得するかどうか
use_init_pose: true

